examples/example_template/example_template.py

```diff
@@ -30,7 +30,8 @@
 # inlined figure.
 
 x = np.linspace(-np.pi, np.pi, 300)
-xx, yy = np.meshgrid(x, x)
+y= np.linspace(-np.pi, np.pi, 300)
+xx, yy = np.meshgrid(x, y)
```

Comment:
Can I ask why we need this change?

Reply:
Yes, as linspace can be coined as Lin i.e linear + space. We are here plotting 2D plot as an example, so it's useful and more generalized way to put array on X,Y.

Reply:
I mean that is true. However, since the values are the same for x and y, I don't see the need for this change.